src/types.ts

```diff
@@ -354,6 +354,36 @@ export interface CalculateMetaTransactionQuoteParams extends SwapQuoteParamsBase
 
 /** End /swap types */
 
+/** Start /rfqm types */
```

Comment:
Let's put these in the RfqmService file, or perhaps in the handler if the same params/response are parsed/returned from there.